rtos/ThisThread.cpp

```diff
@@ -138,4 +138,13 @@ osThreadId_t ThisThread::get_id()
     return osThreadGetId();
 }
 
+const char *get_name()
+{
+    osThreadId_t id = osThreadGetId();
+    if(id == 0) {
```

Comment:
Formatting. Please use `NULL` - CMSIS-RTOS documents that it is a pointer, and says "NULL" for errors.